reactive/kotlinx-coroutines-reactive/src/Await.kt

```diff
@@ -9,114 +9,135 @@ import org.reactivestreams.Publisher
 import org.reactivestreams.Subscriber
 import org.reactivestreams.Subscription
 import java.lang.IllegalStateException
-import java.util.*
+import kotlin.NoSuchElementException
```

Comment:
We use star imports everywhere in our code style (and we're aware it is not aligned with an official recommended code style)

Reply:
Yep, I have that enabled in the config. Seems like Idea is being smart here and detecting that another code style is used in the file already.